Storage/SQL/BrowserSchema.swift

```diff
@@ -1420,6 +1420,17 @@ open class BrowserSchema: Schema {
                 return false
             }
         }
+        
+        if from < 41 && to >= 41 {
+            // As a part of the appservices tabs component integration, remove tabs records from the
```

Comment:
@mhammond I think in the case of a rollback it might be necessary to also delete the client record with the ID value of the FxA ID from the server. If so, I'll add that to this comment.